app/src/main/java/fr/free/nrw/commons/nearby/fragments/NearbyParentFragment.java

```diff
@@ -1861,46 +1943,52 @@ public boolean onItemLongPress(int index, OverlayItem item) {
      *                          locations.
      */
     private void addMarkersToMap(List<BaseMarker> nearbyBaseMarkers) {
-        ArrayList<OverlayItem> items = new ArrayList<>();
         for (int i = 0; i < nearbyBaseMarkers.size(); i++) {
             Drawable icon = ContextCompat.getDrawable(getContext(),
-                getIconFor(nearbyBaseMarkers.get(i).getPlace(), false, isDarkTheme));
+                getIconFor(nearbyBaseMarkers.get(i).getPlace(), false));
             GeoPoint point = new GeoPoint(
                 nearbyBaseMarkers.get(i).getPlace().location.getLatitude(),
                 nearbyBaseMarkers.get(i).getPlace().location.getLongitude());
-            OverlayItem item = new OverlayItem(nearbyBaseMarkers.get(i).getPlace().name,
-                containsParentheses(nearbyBaseMarkers.get(i).getPlace().getLongDescription())
-                    ? getTextBetweenParentheses(
-                    nearbyBaseMarkers.get(i).getPlace().getLongDescription())
-                    : nearbyBaseMarkers.get(i).getPlace().getLongDescription(),
-                point);
-            item.setMarker(icon);
-            items.add(item);
-        }
-        ItemizedOverlayWithFocus overlay = new ItemizedOverlayWithFocus(items,
-            new OnItemGestureListener<OverlayItem>() {
-                @Override
-                public boolean onItemSingleTapUp(int index, OverlayItem item) {
-                    final Place place = nearbyBaseMarkers.get(index).getPlace();
+            Marker marker = new Marker(binding.map);
```

Comment:
Note: I created https://github.com/commons-app/apps-android-commons/issues/5780 about factorization, not your fault, hopefully someone else will factorize later.